directory/cli/src/group.py

```diff
@@ -117,9 +205,9 @@ def remove_member(group_name, users):
         )
     ]
 
-    for command in wrapper_commands:
-        group.add_command(command)
-
+    if utils.advanced_mode_enabled():
```

Comment:
You're probably already aware of this, but as discussed back in https://github.com/alces-software/userware/pull/72/files/6713c525c6cc4a6dbf846bba87c97c8d23b11cfb#r236808603 it could now be worth seeing if we can split out modules for the simple, advanced, and shared parts of both the user and group commands, to make this separation clearer and have less conditionals for defining commands depending on the mode.